scipy/__init__.py

```diff
@@ -128,3 +121,14 @@
     from scipy._lib._testutils import PytestTester
     test = PytestTester(__name__)
     del PytestTester
+
+    # This makes "from scipy import fft" return scipy.fft, not np.fft
+    del fft
+    from . import fft
+
+    # The existence of the scipy.fft function breaks doc builds
+    import os as _os
+    if _os.environ.get('_SCIPY_BUILDING_DOC') == 'True':
+        del fft
+        __all__.remove('fft')
```

Comment:
If `scipy.fft` isn't a function anymore then this shouldn't be needed.

Reply:
Unfortunately I think we need to wait for #10290

Reply:
I mean that if `scipy.fft` is always the module then there's no reason to do `del fft` here as it shouldn't confuse sphinx.

Reply:
Thanks @peterbell10 that fixed the Sphinx problem!